components/camera/replaypcd/replaypcd.go

```diff
@@ -236,7 +237,7 @@ func (replay *pcdCamera) NextPointCloud(ctx context.Context) (pointcloud.PointCl
 	defer cancelTimeout()
 	replay.downloadBatch(ctxTimeout)
 	if ctxTimeout.Err() != nil {
-		return nil, errors.Wrap(ctxTimeout.Err(), "failed to download batch")
+		return nil, errors.Join(ctxTimeout.Err(), errors.New("failed to download batch"))
```

Comment:
@JohnN193 is the format of these errors important? If so I'll revert the replay cameras.

Reply:
none of these errors should be important!